client/state/jetpack-connect/test/selectors.js

```diff
@@ -19,13 +19,16 @@ import {
 	getSessions,
 	getSiteIdFromQueryObject,
 	getSSO,
+	getUserAlreadyConnected,
 	hasExpiredSecretError,
 	hasXmlrpcError,
 	isCalypsoStartedConnection,
 	isRedirectingToWpAdmin,
 	isRemoteSiteOnSitesList,
 } from '../selectors';
 
+const jestExpect = global.expect;
```

Comment:
Now there's a good reason to migrate all these tests to use `jest` too 😉

Reply:
😀 Looking forward to cleaning that up in a rebase 💅